packages/devtools_app/test/deep_link_vlidation/deep_links_screen_test.dart

```diff
@@ -48,7 +48,7 @@ final domainErrorlinkData = LinkData(
   domain: 'www.google.com',
   path: '/',
   os: {PlatformOS.android, PlatformOS.ios},
-  domainErrors: [DomainError.existence],
+  domainErrors: [ AndroidDomainError.existence],
```

Comment:
formatting. If you turn on format on save, your IDE will take care of automatic formatting for you.